test/e2e/prometheus_test.go

```diff
@@ -276,7 +279,11 @@ func testMetricsFromPrometheusServer(t *testing.T, data *TestData, prometheusJob
 	// Create a map of all the metrics which were found on the server
 	testMap := make(map[string]bool)
 	for _, metric := range output.Data {
-		testMap[metric["__name__"]] = true
+		name := metric["__name__"]
+		if strings.HasSuffix(name, "_bucket") {
```

Comment:
Nit: Actually "if" is not required as strings.TrimSuffix returns the string unchanged if the specified suffix is not present.

Reply:
Thank you, I removed 'if' to make it simple.